### PR TITLE
Release version 5.0.3

### PR DESCRIPTION
**Pull request by Bento007**
## Reason for Change

- #877 

## Changes

- allow X_spatial

## Notes for Reviewer

### CODE REVIEW

File: .bumpversion.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 5.0.2
+current_version = 5.0.3-rc.0
```
Comment (nayib-jose-gloria) [inline .bumpversion.cfg:2]: should we remove the rc, or keep it and pin that version in data-portal?

**Review by nayib-jose-gloria — COMMENTED** [review]
looks good! 1 question re: version with rc

Also, mind building this locally + validating against one of the spatial datasets?

**Review by nayib-jose-gloria — APPROVED** [review]
actually i had a spatial downloaded already so I built and validated against it. Looks good from an engineering POV, but I still get this error:

`ERROR: uns['spatial'][library_id]['images']['hires'] must be of type numpy.uint8, it is float32.
`
I think the conclusion was that curators were ok to fix this on their ends, right? If we confirm that, we're ready to go.

**Comment by jahilton** [comment]
>I think the conclusion was that curators were ok to fix this on their ends, right? If we confirm that, we're ready to go.

Yes, we are prepared to start curating the images to uint8 instead of float32
and we plan for migrate.py to convert anything already in the portal

**Pull request merged**